infrastructure-playbooks/cephadm.yml

```
@@ -143,10 +143,16 @@
         state: directory
 
     - name: bootstrap the new cluster
-      command: "{{ cephadm_cmd }} bootstrap --mon-ip {{ _current_monitor_address }} --skip-pull --skip-monitoring-stack {{ '--initial-dashboard-user ' + dashboard_admin_user + ' --initial-dashboard-password ' + dashboard_admin_password if dashboard_enabled | bool else '--skip-dashboard' }}"
-      changed_when: false
-      environment:
-        CEPHADM_IMAGE: '{{ ceph_docker_registry }}/{{ ceph_docker_image }}:{{ ceph_docker_image_tag }}'
+      cephadm_bootstrap:
+        mon_ip: "{{ _current_monitor_address }}"
+        image: "{{ ceph_docker_registry }}/{{ ceph_docker_image }}:{{ ceph_docker_image_tag }}"
+        docker: "{{ true if container_binary == 'docker' else false }}"
+        pull: false
+        dashboard: "{{ true if dashboard_enabled | bool else false }}"
```

why do we need this given there's a default in ceph-defaults for `dashboard_enabled` ?

```suggestion
        dashboard: "{{ dashboard_enabled }}"
```